CONTRIBUTING.md

```diff
@@ -14,70 +14,92 @@ In your bug report please include the following
-
-Write a short paragraph describing what you have done in the pull request and which issue it addresses.
-
-From there, one of out team members here at VicRoy will review your pull requestand give you some constructive feedback. Once it all looks good to go, we would be honoured to merge your contribution!
-
-## How to Set Up Your Environment and Run Tests
+After you have set up Android Studio, fork this repo by simply clicking the fork button in the top right of the screen.
 
-All you need is Android Studio installed and set up!
+![Fork Repo button](docImages/image.png)
 
-Follow this link to download Android Studio: https://developer.android.com/studio
+From your new forked repo, simply clone the project to your machine.
 
-And watch this video to see how to set up Android Studio: https://www.youtube.com/results?sp=mAEB&search_query=how+to+set+up+android+studio
+    Copy the repo URL (click the green "Code" button and copy the link)
+    Opeining the command prompt
+    Navigate to the file that you want the project in
+    Enter the command (git clone [repo link])
 
-Once you have opened the project, you can run it by clicking the green triangular play button on the top right of the screen, and the app will open up in your android emulator. Alternitavely, if you have an android phone, you can run the app on your physical device by following these instructions: https://developer.android.com/studio/run/device
+Now you have a personal copy of RondayView that you can work on!
 
 ## Technical Requirements for Contributions
 
 Our backend logic is all in java so please use java and not kotlin!
 
-*****CHANGE THIS PERHAPS
 We also value having a consistent style in our program, this is to minimise confusion and keep everything looking neat and consistent! As such we will be following
 googles style guide for java programs shown at this link: https://google.github.io/styleguide/javaguide.html
 
 Also, our branch documentation style is as follows
 
     Start the branch with one of these tags 
 
-        feat - for feature
-        fix - for bug fix
-        test - for testing additions
-        docs - documentation update
+        feat/ - for feature
+        fix/ - for bug fix
+        test/ - for testing additions
+        refactor/ - for refactoring code (not adding anything new)
+        docs/ - documentation update
+
+    Then a relevant title for your branch all in lower case with words seperated by dashes (-)
+
+Here is an example of a branch name: feat/creating-home-ui
+
+## Keeping Branches Up To Date
+
+In order to prevent your branches getting out of date, it is important to regularly pull changes from the upstream main branch. The instructions to do this via rebasing are detailed below:
+
+1) Sync the main branch on your fork with the main branch from the upstream using this button on your fork
+
+![Synce Fork Button](docImages/img.png)
 
-    Then add a /
+2) On your local main branch, then pull these changes using
```

Comment:
I think in terms of step-by-step instructions, some people may find this sentence a little confusing. Perhaps consider changing the sentence around a bit!